libs/safety_manager.py

```
@@ -180,6 +180,36 @@ class ExecutionSafetyManager:
+		r"/var/\w+",
+		r"/usr/\w+",
+		r"/root/\w+",
+		r"/home/\w+/",
+		r"/proc/\w+",
+		r"/sys/\w+",
+		r"/dev/\w+",
+		r"/boot/\w+",
+		r"/opt/\w+",
+		r"/mnt/\w+",
+		r"/media/\w+",
+	]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)